### ISSUE TITLE
Used Trezor Model T to convert ADA have Ledger Nano S with same seed that shows 0 balance

### ISSUE THREAD

**Issue by JBillzGH** [vacuumlabs/adalite#695]
I connected Trezor Model T to adalite.io and was able to convert the balance to Shelley address for staking and was able to delegate.  I have a Ledger Nano S that uses the same 24 seed words and when I connect the Ledger Nano S to adalite.io it shows a 0 balance.  

I am able to connect both the Trezor and Ledger to myetherwallet.com and see the same eth balance so I know for certain they are using the same 24 words.  However when I connect to myetherwallet.com with the Trezor it defaults to a derivation path of m/44'/60'/0'/0  while the Ledger uses m/44'/60'/0' and this is where the eth balance is on both devices.  Why am I not able to use the Ledger Nano S with adalite.io to see my ADA balance?

Also, since converting the ADA on adalite.io I am no longer able to connect the Trezor Model T with the Yoroi web wallet.  When I try to connect now it gives an error that it can't connect after exporting the public key.

![image](https://user-images.githubusercontent.com/69698520/90295659-6b6f0d00-de3e-11ea-9b7a-e5699f2b1144.png)

**Comment by ppershing** [comment]
Hi.
Trezor and ledger have incompatible mnemonics for Cardano. This is due to
crypto derivation they use. Basically one of them (I don't remember which
one) deviates from Cardano spec and uses bitcoin spec. Note that the
difference is only when derivating root secret key from mnemonic,
afterwards they use the same crypto. This makes them incompatible...

Best regards,
Peter

**Comment by MichalPetro** [comment]
Also - using the same Trezor on AdaLite and on Yoroi should work without any issues.

**Comment by JBillzGH** [comment]
I used to be able to use the Trezor on Yoroi until I used adalite.io to convert the address to a staking address for delegation.  Now when I open Yoroi in Chrome it shows 0 balance with the last transaction being the funds transferred out of the Byron address as expected.  But if I try to add a new wallet and connect to Trezor so I can see the Shelley address I get the error you see in the image above in original post.  What am I doing wrong and how can I access the funds in Yoroi for the address adalite.io created?
